app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -33,10 +33,8 @@
   }
 }
 
-@include govuk-grid-row($class: "gem-c-feedback__grid-row");
-
 .gem-c-feedback__column-two-thirds {
-  @include govuk-grid-column(two-thirds, $class: false)
+  @include govuk-grid-column($width: two-thirds)
```

Comment:
Could the feedback component just be updated to use `govuk-grid-column-two-thirds`? It looks like the row class was changed to `govuk-grid-row` in the markup.

Reply:
Yes, wasn't constrained to do it for the row, but still got wiggle room for columns. I personally think it should use both classes from `govuk-frontend`.

Reply:
Updated feedback component to use govuk-frontend classes for layout in 6ab1174